Assets/GothicVR/Scripts/UI/TurnSettingDropdownController.cs

```diff
@@ -21,31 +22,41 @@ void Awake()
         foreach (var item in items)
         {
             dropdown.options.Add(new TMP_Dropdown.OptionData() { text = item });
-
         }
         dropdown.itemText.font = GameData.I.GothicSubtitleFont;
-
         dropdown.onValueChanged.AddListener(delegate { DropdownItemSelected(dropdown); });
         snapTurn = locomotionsystem.GetComponent<ActionBasedSnapTurnProvider>();
         continuousTurn = locomotionsystem.GetComponent<ActionBasedContinuousTurnProvider>();
+
+        dropdown.value = PlayerPrefs.GetInt(turnSettingPlayerPref);
+        DropdownItemSelected(dropdown);
     }
 
     void DropdownItemSelected(TMP_Dropdown dropdown)
     {
         switch (dropdown.value)
         {
-            case 0:
-                snapTurn.enabled = true;
-                continuousTurn.enabled = false;
-                break;
             case 1:
-                snapTurn.enabled = false;
-                continuousTurn.enabled = true;
+                EnableContinuousTurn();
                 break;
+            case 0:
             default:
-                snapTurn.enabled = true;
-                continuousTurn.enabled = false;
+                EnableSnapTurn();
                 break;
         }
     }
+
+    void EnableSnapTurn()
+    {
+        snapTurn.enabled = true;
+        continuousTurn.enabled = false;
+        PlayerPrefs.SetInt("TurnSetting", 0);
```

Comment:
Constant please.

We can also think about a whole file where we only store the Constants.

e.g. 
```c#
public class PlayerPrefConstants
```
Because it will be in the future, that we need to get these keys somewhere else if we stick with PlayerPref.

Reply:
Added to constants manager. done.